spherov2/commands/sensor.py

```diff
@@ -8,11 +8,49 @@
     MotorThermalProtectionStatus, ThermalProtectionStatus
 
 
+class CliffDetectionSensorLocations(IntFlag):
+    FRONT_LEFT = 0b1
+    FRONT_RIGHT = 0b10
+    BACK_LEFT = 0b100
+    BACK_RIGHT 0b1000
+
+
+class CollisionDetectedAxis(IntFlag):
+    X_AXIS = 0b1
+    Y_AXIS = 0b10
+    
+    
+class GyroMaxFlags(IntFlag):
+    MAX_PLUS_X = 0b1
+    MAX_MINUS_X = 0b10
+    MAX_PLUS_Y = 0b100
+    MAX_MINUS_Y = 0b1000
+    MAX_PLUS_Z = 0b10000
+    MAX_MINUS_Z = 0b100000
+    
+ 
+#This is what is known, no idea conversion currently
+#class InfraredSensorLocations(IntFlag):
+    #FRONT_LEFT = 0b11111111 #0xff
+    #FRONT_RIGHT = 0b1111111100000000 #0xff00
+    #BACK_LEFT = 0b11111111000000000000000000000000 #-0x1000000
```

Comment:
This is `0xff000000`, unsigned.

Reply:
Weird that they have FRONT_LEFT < RIGHT but BACK_LEFT > RIGHT, can you double check this value?

Reply:
this is displayed in decompiled apk as -0x1000000 so i assumed it was 0xff000000 for the sake of binary conversion. Going to leave in as this until it's tested and proven inaccurate